runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/Chrome.test.stg

```diff
@@ -74,7 +74,7 @@ LANotEquals(i, v) ::= <%this._input.LA(<i>)!=<v>%>
 
 TokenStartColumnEquals(i) ::= <%this._tokenStartColumn===<i>%>
 
-ImportListener(X) ::= <<var <X>Listener = require('./<X>Listener').<X>Listener;>>
+ImportListener(X) ::= ""
```

Comment:
@ericvergnaud if I recall correctly these files are not needed anymore since we dropped the browser test suite in https://github.com/antlr/antlr4/pull/2755/files